com.unity.render-pipelines.high-definition/Runtime/Lighting/Reflection/HDAdditionalReflectionData.cs

```diff
@@ -27,6 +27,8 @@ public static class HDAdditionalReflectionDataExtensions
         /// be rendered the next time it will influence a camera rendering.
         ///
         /// If the probe don't have a <see cref="HDAdditionalReflectionData"/> component, nothing is done.
+        ///
+        /// Note: A camera is influenced by a probe if the camera's frustum intersects the probe's influence volume.
```

Comment:
Note: If any part of a Camera's frustum intersects a Reflection Probe's influence volume, the Reflection Probe influences the Camera.